sys-process/runit/runit-2.2.0.ebuild

```diff
@@ -50,22 +48,34 @@ src_install() {
 	dobin $(<../package/commands)
 	dodir /sbin
 	mv "${ED}"/usr/bin/{runit-init,runit,utmpset} "${ED}"/sbin/ || die "dosbin"
-	if use split-usr ; then
-		dosym ../etc/runit/2 /sbin/runsvdir-start
+	if use scripts ; then
+		if use split-usr ; then
+			dosym ../etc/runit/2 /sbin/runsvdir-start
+		else
+			dosym ../../etc/runit/2 /sbin/runsvdir-start
+		fi
 	else
-		dosym ../../etc/runit/2 /sbin/runsvdir-start
+		exeinto /sbin
+		newexe "${FILESDIR}"/2-${ver_runit_cfg} runsvdir-start
```

Comment:
> I'd like to keep runsvdir-start still available when USE="-script".

It makes no sense to have `/sbin/runsvdir-start` which differs from user's actual `/etc/runit/2`. Why not just keep it as is, as a symlink to `/etc/runit/2`?